docs: add EDB to the `rag-llm-gitops` validated pattern (#457)
validatedpatterns/docs: content/patterns/rag-llm-gitops/_index.md

## Introduction

> This deployment is based on the `validated pattern framework`, using GitOps for
> seamless provisioning of all operators and applications. It deploys a Chatbot
> application that harnesses the power of Large Language Models (LLMs) combined
> with the Retrieval-Augmented Generation (RAG) framework.
> 
> The application uses either the [EDB Postgres for Kubernetes operator](https://catalog.redhat.com/software/container-stacks/detail/5fb41c88abd2a6f7dbe1b37b)
> (default) or Redis to store embeddings of Red Hat products, running on Red Hat
> OpenShift to generate project proposals for specific Red Hat products.

Reviewer: ```suggestion
This deployment is based on the _validated pattern framework_, using GitOps for
seamless provisioning of all operators and applications. It deploys a Chatbot
application that harnesses the power of Large Language Models (LLMs) combined
with the Retrieval-Augmented Generation (RAG) framework.

The application uses either the [EDB Postgres for Kubernetes operator](https://catalog.redhat.com/software/container-stacks/detail/5fb41c88abd2a6f7dbe1b37b)
(default), or Redis, to store embeddings of Red Hat product documentation, running on Red Hat
OpenShift Container Platform to generate project proposals for specific Red Hat products.
```

- LLM Application augmented with content from Red Hat product documentation.

> - Multiple LLM providers (OpenAI, Hugging Face, NVIDIA)

Reviewer: ```suggestion
- Multiple LLM providers (OpenAI, Hugging Face, NVIDIA).
```

> - Vector Database, such as EDB Postgres for Kubernetes or Redis, to store embeddings of RedHat product documentation

Reviewer: ```suggestion
- Vector Database, such as EDB Postgres for Kubernetes, or Redis, to store embeddings of Red Hat product documentation.
```

> - Monitoring dashboard to provide key metrics such as ratings
> - GitOps setup to deploy e2e demo (frontend / vector database / served models)

Reviewer: ```suggestion
- Monitoring dashboard to provide key metrics such as ratings.
- GitOps setup to deploy e2e demo (frontend / vector database / served models).
```
